File: website/get_quarters.php
<?php
	$db = new SQLite3('first.db');
	$statement = $db->prepare('SELECT q.quarter
	                           FROM quarters q
	                                LEFT JOIN quarters_for_courses qc ON qc.quarter_id = q.quarter_id
	                                LEFT JOIN courses c ON c.course_id = qc.course_id
	                           WHERE c.course_id = :id;');
	$statement->bindValue(':id', $_POST['course_id']);
	$results = $statement->execute();
	$ret_val = [];
	while ($row = $results->fetchArray())
	{
		array_push($ret_val, $row['quarter']);
	}
	echo json_encode($ret_val);

?>

File: website/get_quarters.py
#!/usr/bin/python

import json
import sqlite3
import sys

import dict_factory

conn = sqlite3.connect("../data/courses.db")
conn.row_factory = dict_factory.dict_factory
c = conn.cursor()
results = c.execute('''SELECT q.quarter FROM quarters q
                        	LEFT JOIN quarters_for_courses qc ON qc.quarter_id = q.quarter_id
                        	LEFT JOIN courses c ON c.course_id = qc.course_id
                       WHERE c.course_id = ?''', (sys.argv[1],))
json_results = []
result = results.fetchone()
while result != None:
	json_results.append(result["quarter"])
	result = results.fetchone()

print json.dumps(json_results)
